db/controllers/news-controllers.js

```diff
@@ -0,0 +1,14 @@
+const selectTopics = require("../models/news-models");
+
+const getTopics = (request, response, next) => {
+  const { slug } = request.query;
```

Comment:
make sure you only use this when we get to the query endpoint. Task 2 is only asking for an array of all the topics 🧐